CPT331.Data/OffenceRepository.cs

```diff
@@ -100,15 +101,27 @@ public static List<Offence> GetOffences()
 			return offences;
 		}
 
-        /// <summary>
-        /// Updates an Offence record inside the Event-Guardian database, using the ID and values provided. 
-        /// </summary>
-        /// <param name="id">ID indicating the Offence record to be updated</param>
-        /// <param name="isDeleted">New value for the IsDeleted column</param>
-        /// <param name="isVisible">New value for the IsVisible column</param>
-        /// <param name="name">New value for the Name column</param>
+		/// <summary>
+		/// Exports a list of ReadOnlyDataObject types to a stream.
+		/// </summary>
+		/// <param name="readOnlyDataObjects">The list of ReadOnlyDataObject objects to export.</param>
+		/// <param name="stream">The stream to export the ReadOnlyDataObject list to.</param>
+		protected override void OnExport(List<ReadOnlyDataObject> readOnlyDataObjects, Stream stream)
```

Comment:
This is where a list of objects this repository creates is pushed to the base for handling.